Sources/CreateOptions/ConfigOptions.swift

```diff
@@ -232,6 +232,58 @@ public struct ConfigOptions: ParsableConfiguration {
     public var commentsEnabled: Bool {
         !commentOptions.isEmpty
     }
+    
+    @Option public var package: Package
+    
+    // sourcery: document,
+    /// Options specifically related generated a Swift Package.
+    public struct Package: ParsableConfiguration {
+        public init() { }
+        
+        /// Additional remote Swift Package imports.
+        ///
+        /// <details>
+        /// <summary>Examples</summary>
+        ///
+        /// ```yaml
+        /// package:
+        ///   dependencies:
+        ///   - url: https://github.com/apple/swift-argument-parser
+        ///     product: ArgumentParser
+        ///     rule:
+        ///       exact:
+        ///         version: 1.1.1
+        ///   - url: https://github.com/apple/swift-algorithms
+        ///     product: Algorithms
+        ///     rule:
+        ///       range:
+        ///         from: 1.0.0
+        ///         to: 2.1.0
+        ///   - url: https://github.com/apple/swift-metrics.git
+        ///     product: Metrics
+        ///     rule:
+        ///       closedRange:
+        ///         from: 2.0.0
+        ///         to: 3.1.0
```

Comment:
I can never remember the difference between range or closed range... Could these examples maybe better reflect how you might use them?

```suggestion
        ///   - url: https://github.com/apple/swift-algorithms
        ///     product: Algorithms
        ///     rule:
        ///       range:
        ///         from: 1.0.0
        ///         to: 2.0.0
        ///   - url: https://github.com/apple/swift-metrics.git
        ///     product: Metrics
        ///     rule:
        ///       closedRange:
        ///         from: 2.0.0
        ///         to: 2.9.1
```

I think that is the right way? 😅 So `closedRange` would also pick 2.9.1 if it was available whereas `range` wouldn't pick 2.0.0?

Reply:
Range would be exclusive of the upper bound and closed range inclusive would be inclusive of the upper bound, so yes it works as you think it would.

Reply:
Great thanks. Could we commit the above suggestion to the docs to help distinguish the two a little more in that case please?